__tests__/app.test.js

```diff
@@ -1,20 +1,17 @@
 const endpointsJson = require("../endpoints.json");
-/* Set up your test imports here */
-const seed = require("../db/seeds/seed")
-const db = require("../db/connection")
-const data = require("../db/data/test-data")
-const app = require("../app")
-const request = require("supertest")
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const app = require("../app");
+const request = require("supertest");
 
-/* Set up your beforeEach & afterAll functions here */
-
-beforeEach(() => { 
-  return seed(data)
-})
+beforeEach(() => {
+  return seed(data);
```

Comment:
Now we are asking for information from our database, making sure the database is freshly seeded before each test is really good to see 🥇